packages/esm-outpatient-app/src/queue-patient-linelists/queue-linelist-base-table.component.tsx

```diff
@@ -110,6 +115,7 @@ const QueuePatientBaseTable: React.FC<QueuePatientTableProps> = ({
               kind="ghost"
               renderIcon={(props) => <Filter size={16} {...props} />}
               iconDescription={t('filter', 'Filter (1)')}
+              onClick={() => setShowOverlay(true)}
               size="sm">
               {t('filter', 'Filter (1)')}
```

Comment:
Is this meant to have the (1) or the count should be dynamic?

Reply:
They should be dynamic, i will adress that in another pr